.rubocop_todo.yml

```diff
@@ -40,6 +40,7 @@ Metrics/AbcSize:
   Max: 20
   Exclude:
     - 'lib/vagrant-service-manager/command.rb'
+    - 'plugins/guests/redhat/cap/box_version.rb'
```

Comment:
No @hferentschik , we fallback to 1.8.1 and Virtualbox 5.0.26 (CDK 2.3) due to issues in vagrant 1.9.1 in Windows and other issues reported by QE (also updated in mail).